styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss

```diff
@@ -44,7 +45,7 @@
 
 .hub-hero_title {
   background-color: rgba($white, .45);
-  padding: 20px 20px 0;
+  padding: 20px 20px 8px;
```

Comment:
This effects all of the hub hero patterns. It's not a problem, just making sure it's intentional.